templates/partial/_careers-available-roles.html

```diff
@@ -36,11 +36,9 @@
       </div>
     </div>
   </div>
-  <div class="u-fixed-width p-notification u-no-margin--bottom js-filter__no-results u-hide">
-    <p class="p-notification__response">
-      Sorry, there are no open positions matching your criteria.
-    </p>
-  </div>
+  <h4 class="js-filter__no-results u-hide">
```

Comment:
This header and the one on line 66 should both either be `h2`s or `p`s (with h4/h3 classes if need be), the heirarchy at the moment goes straight from `h1` to whichever one is displayed.